Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/sparkserverlessnode/SparkServerlessADLAccountNode.java

```diff
@@ -80,6 +80,9 @@ protected void refreshItems() throws AzureCmdException {
     protected void loadActions() {
         super.loadActions();
 
+        addAction("Submit Serverless Spark Job", new CosmosSparkServerlessSubmitAction(
+                this, adlAccount, SparkServerlessClusterOps.getInstance().getServerlessSubmitAction()));
```

Comment:
I think we should sync with Wei to make sure whether we need a switch for this feature